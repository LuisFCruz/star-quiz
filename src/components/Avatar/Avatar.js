import React, { Component } from 'react'
import PropTypes from 'prop-types'

export class Avatar extends Component {
  static propTypes = {
    id: PropTypes.string.isRequired,
    className: PropTypes.string
  }

  render() {
    const { id, className } = this.props;
    const image = `/assets/images/characters/${id}.jpg`;
    return (
      <img
        className={className}
        src={image}
        alt=""
      />
    )
  }
}

export default Avatar;
