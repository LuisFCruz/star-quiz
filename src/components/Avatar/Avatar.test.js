import React from 'react'
import renderer from 'react-test-renderer'

import { Avatar } from './Avatar'

describe('<Avatar />', () => {
  const defaultProps = { id: '1', className: ''};
  const wrapper = renderer.create(<Avatar { ...defaultProps } />).toJSON();
  
  test('render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
