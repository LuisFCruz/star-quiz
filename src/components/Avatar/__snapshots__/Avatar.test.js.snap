// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> render 1`] = `
<img
  alt=""
  className=""
  src="/assets/images/characters/1.jpg"
/>
`;
